frontend/src/components/Header/Header.tsx

```diff
@@ -733,7 +733,7 @@ export const Header: React.FC<Props> = ({ fullyIntegrated }) => {
 																		)
 																	},
 															  )}
-														<Divider className="mt-1 mb-0" />
+														<Divider className="mb-0 mt-1" />
```

Comment:
One of the eslint changes was to the tailwind class ordering. This was an autofix as a result of that.